css/styles.css

```diff
@@ -34,15 +35,19 @@ header, footer {
     width: 100%;
     display: none;
 }
-#playerControls {
-    display: none;
-}
-
 #zen-video-title #demo {
     margin-top: 5px;
     font-weight: bold;
     word-wrap: break-word;
 }
+#zen-video-error {
+    text-align: center;
+    margin-top: 5%;
```

Comment:
Sorry, let's change this to 30px actually. We're not really using percents for margins, my mistake